sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -403,9 +425,10 @@ bool _parse_options(int msg_type, gnrc_rpl_instance_t *inst, gnrc_rpl_opt_t *opt
                     break;
                 }
                 ipv6_addr_set_aiid(&pi->prefix, iid.uint8);
-                gnrc_netif_ipv6_addr_add(netif, &pi->prefix, pi->prefix_len,
-                                         GNRC_NETIF_IPV6_ADDRS_FLAGS_STATE_VALID);
-                /* TODO: add to prefix list */
+                gnrc_netif_ipv6_addr_add(netif, &pi->prefix, pi->prefix_len, 0);
```

Comment:
0 instead of valid, so DAD can take place.

Reply:
This doesn't work if no 6LBR is present. Since I expect people that they just want to try it between two nodes (and setting a BR up in the Grenoble IoT-LAB testbed is next to impossible) I keep it as it was for now, but keep the thingy with the lifetime ;-).